src/main/java/com/google/sps/model/AuthenticationVerifier.java

```diff
@@ -14,15 +14,20 @@
 
 package com.google.sps.model;
 
+import com.google.sps.utility.KeyProvider;
 import java.io.IOException;
 import java.security.GeneralSecurityException;
 
 /** Contract for verifying user with Google Sign in API */
 public interface AuthenticationVerifier {
-  // OAuth 2.0 Client ID
-  // TODO: Hide Client ID (Issue #72)
-  String CLIENT_ID = "12440562259-mf97tunvqs179cu1bu7s6pg749gdpked.apps.googleusercontent.com";
```

Comment:
If it's indeed important to hide this value, we might to generate a new one as this value will be public "forever"

Reply:
Agreed, I will do the same for the API key as well.

Reply:
`RESOLVED`